packages/@expo/server/src/runtime/error.ts

```diff
@@ -0,0 +1,76 @@
+export class StatusError extends Error {
+  status: number;
+  body: string;
+
+  constructor(status?: number, body?: { error?: string; [key: string]: any } | Error | string);
```

Comment:
The first type seems redundant, since it's the same as the 3rd one, minus the `options`.

(VSCode doesn't show it to me, only the 2nd and 3rd.)

Reply:
Merged; Happy to address this in a follow-up PR though, if the autocompletion isn't ideal